streampark-common/src/main/java/org/apache/streampark/common/constants/Constants.java

```diff
@@ -51,6 +53,8 @@ private Constants() {
 
     public static final String STREAMPARK_SPARKSQL_CLIENT_CLASS = "org.apache.streampark.spark.cli.SqlClient";
 
+    public static final String STREAMPARK_FLINKCDC_CLIENT_CLASS = "org.apache.streampark.cdc.cli.CDCClient";
```

Comment:
It is advisable to rename the package to `org.apache.streampark.flink.cdc....`.

Reply:
modified.